cli/tests/it/config.rs

```diff
@@ -420,12 +422,25 @@ forgetest_init!(can_detect_lib_foundry_toml, |prj: TestProject, mut cmd: TestCom
     pretty_assertions::assert_eq!(
         remappings,
         vec![
-            "another-lib/=lib/nested-lib/lib/another-lib/src/".parse().unwrap(),
-            "ds-test/=lib/forge-std/lib/ds-test/src/".parse().unwrap(),
-            "forge-std/=lib/forge-std/src/".parse().unwrap(),
-            "nested-lib/=lib/nested-lib/src/".parse().unwrap(),
-            "nested-twice/=lib/nested-lib/lib/another-lib/lib/nested-twice/".parse().unwrap(),
-            "nested/=lib/nested-lib/lib/nested/".parse().unwrap(),
+            // NOTE(onbjerg): For some reason a part of this remapping was not normalized on
+            // Windows
+            format!("{}another-lib/src/", remapping_str("another-lib/", "lib/nested-lib/lib/"))
```

Comment:
Not sure what's going on here... I think it might be because the path contains other remappings? It's a bit hard to follow